libcontainer/specconv/example.go

```diff
@@ -217,5 +217,13 @@ func ToRootless(spec *specs.Spec) {
 	spec.Mounts = mounts
 
 	// Remove cgroup settings.
+	// NOTE: cgroups is supported when the permission is properly configured: https://github.com/opencontainers/runc/pull/1540
+	// The caller can set cgroups configuration after calling ToRootless().
 	spec.Linux.Resources = nil
+	spec.Linux.CgroupsPath = ""
+
+	// Disable unsupported security stuff
+	spec.Process.User.AdditionalGids = nil
+	spec.Process.ApparmorProfile = ""
+	spec.Process.SelinuxLabel = ""
```

Comment:
We can't just disable this stuff like that -- if a user has specified them in the configuration then we have to either fail or try to do what they've asked. If we just ignore it (which is what this change does) then we could lead users into thinking their container has security properties it doesn't.

Also `spec.Process.User.AdditionalGids = nil` will break containers that are using `newgidmap` to allow for `setgroups` and the mapping of multiple users (unless I'm mistaken). The same with `CgroupsPath` -- if a privileged process has set up cgroups for us then we can use them.

Reply:
I'm closing but `AdditionalGids` is not supported either way at the moment even when newgidmap is used

https://github.com/opencontainers/runc/blob/3a079311a7e9afeab4b722616cefd2f9b4129104/libcontainer/init_linux.go#L286-#L294